costSplit/index.html

```diff
@@ -1,36 +1,121 @@
+
+input[type="radio"]:checked + div {
+  display: 'block';
+}
+ 
+ .error {
+   color: red;
+ }
+
+</style>
 
 <body>
-  <section class="container">
-  
+  <main class="container">
     <h2>Cost Spliter</h2>
-    <input type='number' id="amt"/>
-    <input type='number' id="splitCost"/>
-    <button id="btn">Split!!</button>
-    <p id="textField"></p>
-  </section>
-<script>const amountField = document.getElementById("amt");
-  const splitCostField = document.getElementById("splitCost");
-  const button = document.querySelector('button');
-  const textField = document.querySelector("#textField");
-  
-  button.addEventListener('click', () => {
-    let amount = Number(amountField.value);
-    let splitCost = Number(splitCostField.value);
-    
-    const indvAmt = calculateAmount(amount, splitCost);
-
-    const text = `Every person needs to pay ${indvAmt}`
-    textField.innerHTML = text;
-  })
-  
-  const calculateAmount = (amount,splitCost) =>{
-    let indvi_amt = amount/splitCost;
-     return indvi_amt;
-  }
-</script>
+    <section>
+      <!-- amount to split -->
+      <label for='amt'>Amount
+        <input type='number' id="amt" />
+      </label>
+      <!-- choice of how to split -->
+      <div>
+        <label for="splitCost2">Split cost by person</label>
+        <input type="radio" id="splitCost2" name="split-cost" value="byPerson" checked>
+
+        <div class='byPerson'>
+          <label for=people>Number of people<input type='number' id="splitCost" /></label>
+          <button id="btn">Split!!</button>
+          <p id="textField"></p>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <label for="splitCost1">Split cost by percent</label>
+      <input type="radio" id="splitCost1" name="split-cost" value="byPercent">
+
+      <div class='byPercent'>
+        <div>
+          <input type='text' id="name" placeholder="Name" />
+          <input type='number' id="person1" placeholder="%" />
+        </div>
+        <div>
+          <input type='text' id="name" placeholder="Name" />
+          <input type='number' id="person2" placeholder="%" />
+        </div>
+        <button id='percentBtn'>Split</button>
+        <p class='error'></p>
+        <p id='amount1'></p>
+        <p id='amount2'></p>
+      </div>
+    </section>
+
+  </main>
+  <script>
+    const amountField = document.getElementById("amt");
+    const splitCostField = document.getElementById("splitCost");
+    const button = document.querySelector('button');
+    const textField = document.querySelector("#textField");
+
+    button.addEventListener('click', () => {
+      let amount = Number(amountField.value);
+      let splitCost = Number(splitCostField.value);
+
+      const indvAmt = calculateAmount(amount, splitCost);
+
+      const text = `Every person needs to pay ${indvAmt}`
+      textField.innerHTML = text;
+    });
+
+    const amount1 = document.querySelector('#amount1');
+    const amount2 = document.querySelector('#amount2');
```

Comment:
I really like to avoid using variable names like person1, 2 etc.
(I myself is very bad in naming conventions. Can you think of better name?).